internal/cluster/validation_id.go

```diff
@@ -28,14 +28,16 @@ const (
 	IsNtpServerConfigured               = validationID(models.ClusterValidationIDNtpServerConfigured)
 	IsOcsRequirementsSatisfied          = validationID(models.ClusterValidationIDOcsRequirementsSatisfied)
 	IsLsoRequirementsSatisfied          = validationID(models.ClusterValidationIDLsoRequirementsSatisfied)
+	SufficientMemoryAvailable           = validationID(models.ClusterValidationIDSufficientMemoryAvailable)
+	SufficientCPUAvailable              = validationID(models.ClusterValidationIDSufficientCPUAvailable)
 )
 
 func (v validationID) category() (string, error) {
 	switch v {
 	case IsMachineCidrDefined, isMachineCidrEqualsToCalculatedCidr, isApiVipDefined, isApiVipValid, isIngressVipDefined, isIngressVipValid,
 		isClusterCidrDefined, isServiceCidrDefined, noCidrOverlapping, networkPrefixValid, IsDNSDomainDefined, IsNtpServerConfigured:
 		return "network", nil
-	case AllHostsAreReadyToInstall, SufficientMastersCount:
+	case AllHostsAreReadyToInstall, SufficientMastersCount, SufficientMemoryAvailable, SufficientCPUAvailable:
```

Comment:
Just wonder if this shouldn't be as part of "operators" instead of "hosts-data". Because it's actually `operators` related. But I would leave this decision to someone from assisted team.

Reply:
Agreed. I'd also like to hear @gamli75 's feedback on this.